### ISSUE TITLE
I can't fill out that initial sign-up form, which ghosts out the rest of the page...

### ISSUE THREAD

**Issue by PenelopeSilke** [MrMeapify/sleepychat#47]
...because for some reason on my macbook I can't scroll down to the end of it.

**Comment by ElysianTail** [comment]
Have you tried using a scroll wheel? It is a known issue that the window's
scroll bar doesn't work, but the dialog can still scroll from a wheel.
Secondly, you can just hit enter from the username field, and that should
allow you to enter chat.

**Comment by PenelopeSilke** [comment]
Thank you for your quick response!  Somehow it worked when I tried it again, <shrug>.  Thank you again.

**Comment by JacquesTreatment** [comment]
I don't have access to a "scroll wheel" with this trackball.  It'd be nice to try out the matchmaker thing, but until this is fixed, I cannot.

**Comment by ElysianTail** [comment]
A second alternative is to full out what you can and simply press enter
while the username field is selected.

**Comment by JacquesTreatment** [comment]
Yes, that lets me into the main room, but not to the other function.  I believe I was able to use that once or twice, before I had to switch computers.

**Comment by MrMeapify** [comment]
There's some CSS that needs to be changed for that form, but as a bandaid
you can zoom out your browser.

**Comment by ElysianTail** [comment]
Lastly, if you can zoom the page out, the whole dialog should be visible. I
know it's an annoying bug, but it's a part of the theming system and
they're taking their sweet time fixing it.
